### ISSUE TITLE
[Feature][JDBC Sink] Using Kafka to Implement Heterogeneous Database CDC

### ISSUE THREAD

**Issue by EngonVHKxZ** [apache/seatunnel#5900]
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

The Source component of Kafka has an attribute called format. I noticed that it allows for receiving jsons from Debezium, while SeaTunnel allows reading Kafka data in a stream format. Therefore, theoretically, I can read changes to data in the table through FlinkCDC or Debezium and write them to Kafka.
There is also a parameter in JDBC's Sink that allows for automatic generation of SQL statements, which is generate_sink_sql setting it to true will generate SQL insert statements.
If both conditions are met simultaneously, is it possible to convert Debezium's JSON file into the corresponding DML SQL?

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by hailin0** [comment]
Yes, this feature is currently available

**Comment by EngonVHKxZ** [comment]
> Yes, this feature is currently available

Really? After testing, this feature is not available in version 2.3.3. Does this feature only have a master branch? I must give it a try now

**Comment by EngonVHKxZ** [comment]
> Yes, this feature is currently available

Really? After testing this feature is not available in version 2.3.3. Does this feature only have a master branch? I must give it a try now

**Comment by EngonVHKxZ** [comment]
> Yes, this feature is currently available

My usage method is as follows: using FlinkCDC to collect DML operations from Oracle and input them into Kafka, SeaTunnel uses streaming to read CDC operations from Kafka.

****

The following is the data format I saved to Kafka

```
{
    "before": {
        "ID": "0A920E5F701A275DE06011AC020000E0",
        "CODE": "1",
        "NAME": "1",
        "VERSION": "1",
        "SOURCE": "1",
        "VALID_FLAG": 1,
        "SORT_NO": 1,
        "CREATE_OPERATOR": 1,
        "MODIFY_OPERATOR": 1,
        "REMARK": "1"
    },
    "after": {
        "ID": "0A920E5F701A275DE06011AC020000E0",
        "CODE": "2",
        "NAME": "1",
        "VERSION": "1",
        "SOURCE": "1",
        "VALID_FLAG": 1,
        "SORT_NO": 1,
        "CREATE_OPERATOR": 1,
        "MODIFY_OPERATOR": 1,
        "REMARK": "1"
    },
    "source": {
        "version": "1.9.7.Final",
        "connector": "oracle",
        "name": "oracle_logminer",
        "ts_ms": 1700881435000,
        "snapshot": "false",
        "db": "HELOWIN",
        "sequence": null,
        "schema": "LOGMINER_USER",
        "table": "TEST_DICT",
        "txId": "03000b00ff040000",
        "scn": "3671616",
        "commit_scn": "3671617",
        "lcr_position": null,
        "rs_id": null,
        "ssn": 0,
        "redo_thread": 1
    },
    "op": "u",
    "ts_ms": 1700881435880,
    "transaction": null
}
```

***
Here is my configuration file

```
env {
  execution.parallelism = 1
  job.mode = "STREAMING"
}
source {
  Kafka {
    format = debezium-json
    topic = "cdc1"
    bootstrap.servers = "192.168.4.18:9092"
    kafka.config = {
      client.id = client_1
    }
  }
}
sink {
  Jdbc {
      url="jdbc:oracle:thin:@192.168.26.129:1521:helowin",
      driver="oracle.jdbc.OracleDriver",
      user="logminer_user",
      password="logminer_pwd",
      generate_sink_sql=true,
      database="HELOWIN",
      table="LOGMINER_USER.TEST_DICT"
  }
}
```
***
SQL statements printed in the SeaTunnel console

```
INSERT INTO "LOGMINER_USER"."TEST_DICT" ("content") VALUES (?)
```

**Comment by EngonVHKxZ** [comment]
> > Yes, this feature is currently available
> 
> Really? After testing this feature is not available in version 2.3.3. Does this feature only have a master branch? I must give it a try now

This feature is not available in the main branch either
